process-controller/src/main/java/org/jboss/as/process/ManagedProcess.java

```diff
@@ -200,7 +200,6 @@ void doStart(boolean restart) {
         try {
             process = builder.start();
         } catch (IOException e) {
-            e.printStackTrace();
             processController.operationFailed(processName, ProcessMessageHandler.OperationType.START);
             log.failedToStartProcess(processName);
```

Comment:
Having info why the process failed to start might be useful => shouldn't the info from the IOException stay logged somewhere?